### DISCUSSION TITLE
Modify embedded URL regex to match self-hosted domains as well

### DISCUSSION THREAD

**Discussion by intradeus** [outline/outline#3577] [Ideas]
Wouldn't it be nice if we could embed self-hosted webapps as well ? 

For example, I have a self-hosted instance of Tldraw, but the regex in the [Tldraw.tsx](https://github.com/outline/outline/blob/3b62c7620752df4450048b46aef1a10fc2e13d36/shared/editor/embeds/Tldraw.tsx) file is limited to : 
` "https?://www.tldraw.com/r/(.*)$" `

If I create a PR that admits URLs matching `new RegExp("https?://*tldraw*.*/r/(.*)$")]; `, this way you could self-host 'tldraw.yourdomain.ca' and could embed it into outline, would this PR be accepted ? Or is it out of scope / brings security risks ?

The limit is that self-hosted instances would NEED to have 'tldraw', 'airtable' or else, in the domain, in order to match the regex.

**Comment by Limezy** [comment]
It would be nice indeed. But for tldraw, as far as I have understood, the app itself is open source but not the multiplayer server. Multiplayer is needed if you want data to persist and different users to be able to interact... 

For the security risk, I belive the best would be to let users set some "custom" integrations in the settings, whith a little form to upload the icon, a little field to put their own regex for example ?
